libs/blocks/tabs/tabs.css

```diff
@@ -264,6 +265,59 @@
   height: 24px;
 }
 
+.tabs .tab-paddles {
+  position: relative;
+  z-index: 3;
+}
+
+.tabs .tab-paddles .paddle {
+  position: absolute;
+  width: 32px;
+  height: 48px;
+  bottom: 0;
+  background: var(--tabs-active-bg-color);
+  border: 0;
+  padding: 14px 0 12px;
+  display: flex;
+  align-items: center;
+}
+
+.tabs .tab-paddles .paddle:hover {
+  cursor: pointer;
+}
+
+.tabs .tab-paddles .paddle:disabled {
+  box-shadow: 0 -1px 0 inset var(--tabs-border-color);
+  background: var(--tabs-list-bg-gradient);
+}
+
+.tabs .tab-paddles .paddle-left {
+  box-shadow: 0 -1px 0 inset var(--tabs-border-color), 4px 0 4px -2px var(--tabs-paddle-bs-color);
+  left: 0;
+}
+
+.tabs .tab-paddles .paddle-right {
+  box-shadow: 0 -1px 0 inset var(--tabs-border-color), -4px 0 4px -2px var(--tabs-paddle-bs-color);
+  right: 0;
+}
+
+.tabs .tab-paddles .paddle svg {
+  opacity: 100;
```

Comment:
I think this should be either `1` or `100%` - https://developer.mozilla.org/en-US/docs/Web/CSS/opacity#alpha-value

Reply:
Updated